templates/zerver/emails/email.css

```diff
@@ -52,6 +52,12 @@ table.layout > tbody > tr > td {
     padding: 10px;
 }
 
+/* This class is used to ensure that the images' width
+   does not exceed the actual email layout width. */
+img.responsive-width {
+    max-width: 100%;
+}
+
```

Comment:
I preferred not to use this widely on images and use class because we don't have these issues with the other images, and it's safer to fix just these two and not break the others.